test/aida/bot_test.exs

```diff
@@ -853,6 +927,31 @@ defmodule Aida.BotTest do
       assert error_log.skill_id == ld_skill |> Aida.Skill.id()
       assert error_log.message == "Could not find attribute named 'bar'"
     end
+
+    test "when post_notification_message fails", %{bot: bot, session: session} do
+      forward_messages_id = Ecto.UUID.generate()
+      session = session |> Session.put(".forward_messages_id", forward_messages_id)
+
+      response = {:ok, %HTTPoison.Response{status_code: 500}}
+
+      with_mock HTTPoison,
+        post: fn _url, _body -> response end do
+        Message.new("a message", bot, session) |> Bot.chat()
+
+        [error_log] = ErrorLog |> Repo.all()
+        assert error_log.bot_id == bot.id
+        assert error_log.session_id == session.id
+
+        extra = %{
+          post_url: "#{bot.notifications_url}/messages/#{forward_messages_id}",
+          post_body: %{type: "text", direction: "uto", content: "a message"},
+          response: response
+        }
+
+        assert error_log.message ==
+                 "Error posting notification message to remote endpoint #{inspect(extra)}"
```

Comment:
I would prefer to avoid the `inspect()` in the error logs. This is supposed to be a human readable message, not an error dump. You could take a look at `Aida.ErrorHandler` which creates a Sentry log, then logs the error in the standard log, and then opens an error log with the human readable details.

Maybe we can add some method there to handle error logs for something else besides exceptions.

Reply:
I think I've got it. I'll add some method to `Aida.ErrorHandler`, using `Logger.error` for the standard error and `ErrorLog.write` for a human readable one. Please let me know if something doesn't feel right. Thanks!